provisioner/plain-v0/README.md

```diff
@@ -1,3 +1,197 @@
-# plain-v0
+# Plain-v0 Provisioner
 
-A concrete Provisioner implementation for "plain" k8s bundles.
+## Summary
+
+The `plain-v0` provisioner is a core rukpak provisioner that knows how to interact with `plain+v0` format bundles.
+The `plain+v0` bundles, or plain bundles, are simply container images containing a set of static kubernetes YAML
+manifests in a given directory. For example, below is an example Dockerfile that builds a plain `plain+v0` bundle from a
+directory containing static manifests.
+
+```dockerfile
+FROM scratch
+LABEL core.rukpak.io.bundle.mediatype.v1=plain+v0
+LABEL core.rukpak.io.bundle.manifests.v1=/manifests
+LABEL core.rukpak.io.bundle.package.v1=my-package
+LABEL core.rukpak.io.bundle.version.v1=0.0.1
+COPY ./manifests /manifests
+```
+
+where the given `manifests` directory contains the kubernetes resources required to deploy an application, for example:
+
+```
+manifests
+├── 00_namespace.yaml
+├── 01_cluster_role.yaml
+├── 02_role.yaml
+├── 03_serviceaccount.yaml
+├── 04_cluster_role_binding.yaml
+├── 05_role_binding.yaml
+└── 06_deployment.yaml
```

Comment:
This is _currently_ misleading since the apply ordering is not preserved. The plain-v0 provisioner currently delegates to helm's defined ordering.

Reply:
Yes, I removed the number prefix so as not to be misleading.